tutorials/install-docker-ubuntu-jammy-jellyfish/index.mdx

```diff
@@ -0,0 +1,85 @@
+---
+
+Docker is a powerful and widely-used open-source platform that enables developers to automate the deployment, scaling, and management of applications in a consistent and efficient manner.
+
+The tool allows you to run software in a standardized environment called a "container" that encapsulates an application along with all its dependencies and configurations.
+
+Docker Engine is the core component of the [Docker platform](https://docs.docker.com/get-started/overview/#docker-architecture). It is the runtime that allows you to create, run, and manage Docker containers. Docker Engine is responsible for building, running, and distributing containers. It includes the Docker daemon, a long-running process responsible for managing Docker objects like images, containers, volumes, and networks. The main components that you install by following this tutorial are:
+
+* **Docker daemon**: The Docker daemon is a background service responsible for managing Docker containers, images, networks, and volumes. It listens for Docker API requests and handles container lifecycle operations like starting, stopping, and monitoring containers.
+* **Docker CLI**: The Docker Command Line Interface (CLI) is a tool that allows you to interact with Docker. It provides a set of commands to manage containers, images, networks, volumes, and other Docker resources.
+* **Docker Compose**: Docker Compose is a tool that allows you to define and manage multi-container applications using a YAML file. It simplifies the process of defining the services, networks, and volumes required for your application stack.
+* **Containerd**: Containerd is an open-source container runtime that serves as the industry-standard container runtime interface (CRI) between higher-level container management systems (such as Docker and Kubernetes) and the low-level kernel features responsible for running containers.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key/)
+  - You have an Instance running Ubuntu Jammy Jellyfish (22.04 LTS)
+  - You have sudo privileges or access to the root user.
+</Message>
+
+## Installing Docker Engine on Ubuntu Jammy Jellyfish (22.04 LTS)
+
+Prior to installing Docker Engine on a new host machine, you must configure the Docker repository. Once the repository is set up, you can proceed to install and keep Docker updated directly from the repository.
+1. Log in to your Instance using SSH:
+    ```
+    ssh root@<YOUR_INSTANCE_IP>
+    ```
+2. Update the `apt` package cache and upgrade the software already installed on your Instance to the latest version available in the repository:
+    ```
+    apt update && apt upgrade -y
+    ```
+
+### Setting up the Docker repository
+
+1. Install packages that are required to download packages using `apt` via a HTTPS connection.
```

Comment:
maybe add "to download the required packages for Docker" or something like that to explain a bit more ?

Reply:
```suggestion
1. Install packages that are required to download the required packages for Docker using `apt` via an HTTPS connection.
```